### ISSUE TITLE
make tests pass on Windows

### ISSUE THREAD

**Issue by jurko-gospodnetic** [numirias/pytest-json-report#65]
Currently there is one failing test on Windows:
```
def test_report_keys(num_processes, make_json):
    data = make_json()
    keys = set([
        'created', 'duration', 'environment', 'collectors', 'tests', 'summary',
        'root', 'exitcode'
    ])
    if num_processes > 0:
        # xdist only reports failing collectors
        keys.remove('collectors')
    assert set(data) == keys
    assert isinstance(data['created'], float)
    assert isinstance(data['duration'], float)
    assert data['root'].startswith('/')
    assert data['exitcode'] == 1
```
because it checks that `data[root]` starts with an `'/'` - and I expect we should instead check whether it's an absolute path using `os.path.isabs()`

**Comment by jurko-gospodnetic** [comment]
fixed in pull request #66

**Comment by numirias** [comment]
Thanks! The project doesn't currently aim to guarantee Windows compatibility, but I'll happily take patches like this one.
